.github/workflows/api-check.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           # We need nightly for cargo-public-api to get the API output.
-          toolchain: nightly
+          toolchain: nightly-2023-01-04
```

Comment:
If we lock to a Rust toolchain version, we'd better lock to a `cargo-public-api` version as well (in the `cargo install` below). Otherwise, when `cargo-public-api` upgrades the toolchain version, the check will fail again.

Reply:
Oh right. Yep my bad.